charts/airflow/README.md

```diff
@@ -758,6 +758,15 @@ serviceAccount:
 <hr>
 </details>
 
+### How to use `KubernetesExecutor` and `CeleryKubernetesExecutor`?
```

Comment:
I was thinking we should give an example of a values.yaml which will enable `KubernetesExecutor`, (and probably a separate one for `CeleryKubernetesExecutor`, as that needs all the normal Celery Stuff).


For example,
```yaml
airflow:
  executor: KubernetesExecutor

  kubernetesPodTemplate:
    ...

workers:
  enabled: false

redis:
  enabled: false
```

Reply:
Working on that next, I will squash this PR down to one commit when ready or would you like to do that on merge ?

Reply:
Made suggested updates. Would prefer to make the sample values.yaml a separate PR if that is acceptable.